delete_libvirt_vms.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: kvm
```

Comment:
Why not use the existing "helper" host?

Reply:
I just prefer to directly logging in to the kvm host from the client by specifying the host in the master playbook rather than specifying the Libvirt connection URI as follows because it's easy to follow. The other reason is that inputting the root password is required several times when I tried the Libvirt connection.

```
libvirt:
 qemu+ssh://root@192.168.150.1/system
```

Reply:
I had a bit of misunderstanding and I found that the KVM host isn't necessarily set int the inventory file. See https://github.com/noseka1/openshift-auto-upi/pull/22#discussion_r493195990